lib/composer/package/loader/project_attributes_loader.rb

```diff
@@ -0,0 +1,56 @@
+module Composer

```

Comment:
If you do `Gitlab::Composer`, you need to reference the original `Composer` as `::Composer`, where `::` is the "root namespace". Just `Composer` will refer to `Composer` in the current namespace, which is `Gitlab` since you're inside `module Gitlab`.